crates/oxc_parser/Cargo.toml

```diff
@@ -45,3 +45,5 @@ pico-args = { workspace = true }
 [features]
```

Comment:
I guess we need to add a `default` feature with `regex` turned on by default.

Reply:
Doing this makes it hard to disable the `oxc_parser/regex` feature in the `oxc` umbrella crate, so that `oxc/regex` can control it instead. Using `{ workspace = true, default-features = false }` in `oxc` errors.